pkg/ccl/backupccl/targets.go

```diff
@@ -517,11 +521,17 @@ func getRelevantDescChanges(
 		if _, ok := interestingIDs[change.ID]; ok {
 			interestingChanges = append(interestingChanges, change)
 		} else if change.Desc != nil {
-			if table := sqlbase.TableFromDescriptor(change.Desc, hlc.Timestamp{}); table != nil {
+			desc := change.Desc
+			if table := sqlbase.TableFromDescriptor(desc, hlc.Timestamp{}); table != nil {
 				if _, ok := interestingParents[table.ParentID]; ok {
 					interestingIDs[table.ID] = struct{}{}
 					interestingChanges = append(interestingChanges, change)
 				}
+			} else if typ := desc.GetType(); typ != nil {
```

Comment:
ditto from above, especially because we are saying tableFromDescriptor with an empty timestamp